src/components/dialog/dialog-container.ts

```diff
@@ -0,0 +1,70 @@
+import {Component, ComponentRef, ViewChild, AfterViewInit} from '@angular/core';
+import {BasePortalHost, ComponentPortal, TemplatePortal} from '../../core/portal/portal';
+import {PortalHostDirective} from '../../core/portal/portal-directives';
+import {PromiseCompleter} from '../../core/async/promise-completer';
```

Comment:
Shouldn't those imports use the NPM scope package instead?

As @hansl did in 9e308a6fa2d263a117e6a147cfab29387797b46c?